src/SFA.DAS.ProviderRegistrations/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -8,7 +8,7 @@ public static class ServiceCollectionExtensions
     {
         public static IServiceCollection AddDasDistributedMemoryCache(this IServiceCollection services, IConfiguration configuration, bool isDevelopment)
         {
-            var redisConnectionString = configuration.GetSection(ProviderRegistrationsConfigurationKeys.ProviderRegistrations).Get<ProviderRegistrationsSettings>().RedisConnectionString;
+            var redisConnectionString = configuration.GetSection(ProviderRegistrationsConfigurationKeys.RedisConnectionSettings).Get<ProviderRegistrationsSettings>().RedisConnectionString;

```

Comment:
```suggestion
            var redisConnectionString = configuration.GetSection(ProviderRegistrationsConfigurationKeys.RedisConnectionSettings).Get<RedisConnectionSettings>().RedisConnectionString;

```

Reply:
also the RedisConnectionString property should be removed from `ProviderRegisteationSettings.cs`

Reply:
done